blueoil/cmd/init.py

```diff
@@ -228,130 +241,117 @@ def save_config(blueoil_config, output=None):
-        'filter': image_size_filter,
-        'validate': generate_image_size_validate(network_name),
-    }
-    image_size = prompt(image_size_question)
-
-    training_epochs_question = {
-        'type': 'input',
-        'name': 'value',
-        'message': 'how many epochs do you run training (integer):',
-        'input_type': 'integer',
-        'default': '100'
-    }
+    image_size_question = [
+        inquirer.Text(name='value',
+                      message='image size (integer x integer)',
+                      default='128x128',
+                      validate=generate_image_size_validate(network_name))
+    ]
+    image_size = image_size_filter(prompt(image_size_question))
+
+    training_epochs_question = [
+        inquirer.Text(name='value',
+                      message='how many epochs do you run training (integer)',
+                      default='100',
+                      validate=integer_validate)
+    ]
     training_epochs = prompt(training_epochs_question)
 
-    training_optimizer_question = {
-        'type': 'rawlist',
-        'name': 'value',
-        'message': 'select optimizer:',
-        'choices': ['Momentum', 'Adam'],
-        'default': 'Momentum'
-    }
+    training_optimizer_question = [
+        inquirer.List(name='value',
+                      message='select optimizer',
+                      choices=['Momentum', 'Adam'],
+                      default='Momentum')
+    ]
     training_optimizer = prompt(training_optimizer_question)
 
-    initial_learning_rate_value_question = {
-        'type': 'input',
-        'name': 'value',
-        'message': 'initial learning rate:',
-        'default': '0.001'
-    }
+    initial_learning_rate_value_question = [
+        inquirer.Text(name='value',
+                      message='initial learning rate',
+                      default='0.001')
+    ]
     initial_learning_rate_value = prompt(initial_learning_rate_value_question)
 
     # learning rate schedule
-    learning_rate_schedule_question = {
-        'type': 'rawlist',
-        'name': 'value',
-        'message': 'choose learning rate schedule \
-({epochs} is the number of training epochs you entered before):',
-        'choices': list(learning_rate_schedule_map.values()),
-        'default': learning_rate_schedule_map["constant"],
-    }
+    learning_rate_schedule_question = [
+        inquirer.List(name='value',
+                      message='choose learning rate schedule'
+                              ' ({{epochs}} is the number of training epochs you entered before)',
+                      choices=list(learning_rate_schedule_map.values()),
+                      default=learning_rate_schedule_map["constant"])
+    ]
     _tmp_learning_rate_schedule = prompt(learning_rate_schedule_question)
     for key, value in learning_rate_schedule_map.items():
         if value == _tmp_learning_rate_schedule:
             learning_rate_schedule = key
 
+    data_augmentation = {}
```

Comment:
should this be moved up here? it seems it's not used until L383

Reply:
I moved `data_augmentation` out of the if condition because `data_augmentation` won't be declared if the augmentation is disabled. (#881 )

Reply:
ah, i see. I didn't expand enough, my bad. thanks.